src/utils/config/risk-framework.json

```diff
@@ -827,7 +827,7 @@
             "complexityScore": 4,
             "teamKnowledgeScore": 2,
             "criteria": {
-                "nameLike": ["GeistLenderBorrower"],
+                "nameLike": ["GeistLenderBorrower", "GeistMIMLender", "GeistDAILender", "GeistfUSDTLender", "GeistfUSDCLender"],
```

Comment:
We can remove `GeistLenderBorrower` from the list.

Reply:
Resolved in https://github.com/yearn/yearn-watch/pull/281/commits/ed4fdc112d8b4aad30cc53acaf3687df1b95334b